app/assets/javascripts/mobile/mobile_comments.js

```diff
@@ -11,6 +11,8 @@
     initialize: function() {
       var self = this;
 
+      new Diaspora.MarkdownEditor(".comment_box");
```

Comment:
@cmrd-senya What you write seems legit to me, but I'd also like to hear @denschub s opinion on this topic. But _please_ let's do this somewhere else. This is out of scope of this PR. Just open a new code refactor issue or so.